resources/asciidoctor/lib/chunker/breadcrumbs.rb

```diff
@@ -12,9 +12,76 @@ def generate_breadcrumbs(doc, section)
       result = ['<div class="breadcrumbs">']
       result += generate_breadcrumb_links(section).reverse
       result << '</div>'
+      if result[2].to_s.include? 'APM'
+        result[2] = generate_apm_breadcrumbs(doc)
+      end
+      if result[2].to_s.include? 'ECS Logging'
+        result.insert(2, generate_ecslogging_breadcrumbs)
+      end
       Asciidoctor::Block.new doc, :pass, source: result.join("\n")
     end
 
+    def generate_apm_breadcrumbs(doc)
+      title = doc.title
+      short = title.sub(/APM /, '')
+      <<~HTML.strip
+        <span class="breadcrumb-link">
+          <div id="related-products" class="dropdown">
+            <div class="related-products-title">APM:</div>
+            <div class="dropdown-anchor">#{short}<span class="dropdown-icon"></span></div>
+            <div class="dropdown-content">
+              <ul>
+                <li class="dropdown-category">APM</li>
+                <ul>
+                  <li><a id="guide" href="http://localhost:8000/guide/index.html">User Guide</a></li>
+                </ul>
+                <li class="dropdown-category">APM agents</li>
+                <ul>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/android/current/index.html">Android Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/go/current/index.html">Go Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/swift/current/index.html">iOS Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/java/current/index.html">Java Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/dotnet/current/index.html">.NET Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/nodejs/current/index.html">Node.js Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/php/current/index.html">PHP Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/python/current/index.html">Python Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/ruby/current/index.html">Ruby Agent Reference</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/agent/rum-js/current/index.html">Real User Monitoring JavaScript Agent Reference</a></li>
+                </ul>
+                <li class="dropdown-category">APM extensions</li>
+                <ul>
+                  <li><a href="https://www.elastic.co/guide/en/apm/lambda/current/index.html">Monitoring AWS Lambda Functions</a></li>
+                  <li><a href="https://www.elastic.co/guide/en/apm/attacher/current/index.html">Attacher</a></li>
+                </ul>
+              </ul>
+          </div>
+        </div>
+        <span class="chevron-right">›</span>
```

Comment:
This is an interesting bug that I can't quite figure out. Selecting top-level heading doesn't show in the breadcrumbs. Have to select second level to get it to show

<img width="945" alt="Screenshot 2023-02-15 at 9 55 59 AM" src="https://user-images.githubusercontent.com/5618806/219113153-bf9d5aa9-a57b-4bf1-b0c0-9de5674ec338.png">


<img width="831" alt="Screenshot 2023-02-15 at 9 56 04 AM" src="https://user-images.githubusercontent.com/5618806/219113078-fbdc2a90-c914-42f5-a6d7-c662dc7b3750.png">

Reply:
oops didn't mean to comment in the file, sorry

Reply:
Ah! That's not the bug. The bug is that the chevron appears when it shouldn't.

Reply:
This is a complication of replacing instead of inserting

Reply:
Might have to test the length of the array. If length of the array is greater than 3, include chevron

Reply:
Yea I thought that was strange at first too, but that's actually how it works in production now (we just have to hide that last >).

**Production:**

<img width="957" alt="Screenshot 2023-02-15 at 12 13 43 PM" src="https://user-images.githubusercontent.com/10479155/219117098-26adf978-aca7-428c-ba50-f936b5a4510c.png">


**Preview:**

<img width="1055" alt="Screenshot 2023-02-15 at 12 14 17 PM" src="https://user-images.githubusercontent.com/10479155/219117125-6d3c4cc5-107b-454e-aec5-8079675c1d8b.png">

Reply:
I guess the question is: do breadcrumbs usually include the title of the current page or is that assumed because it's directly under the breadcrumbs? I guess it would be silly to add a link to the current page in the breadcrumbs...

Reply:
>Might have to test the length of the array. If length of the array is greater than 3, include chevron

OR add a chevron _before_ each item unless it's the manually-built, first item "Elastic Docs". That seems less confusing to me at least.

Reply:
> OR add a chevron before each item unless it's the manually-built, first item "Elastic Docs". That seems less confusing to me at least.

I think this will be very complicated. The array is built and then reversed